task/04-arrays-tasks.js

```diff
@@ -536,7 +568,16 @@ function selectMany(arr, childrenSelector) {
  *   [[[ 1, 2, 3]]], [ 0, 0, 1 ]      => 2        (arr[0][0][1])
  */
 function getElementByIndexes(arr, indexes) {
-    throw new Error('Not implemented');
+   let new_arr = new Array();
+   let arr_index = 0;
+   indexes.map(each_el => {
+   if (each_el != 0) arr_index = each_el;
```

Comment:
"operator !== is always preferable
sometimes ''!=' may cause an error
(check at console :
'0' != 0
and '0' !== 0)"

Reply:
> Why did you marked this PR as ready for review?
> You should implement all notes from above
I think i did it, or not at all? in my repository all tasks are fixed